partcad/src/partcad/runtime_python.py

```diff
@@ -94,7 +95,6 @@ def once(self):
         with self.lock:
             if not self.initialized:
                 # Preinstall the most common packages to avoid race conditions
-                # TODO(clairbee): Lock the entire runtime instead
```

Comment:
I'm using `# TODO: @openvmp: ...` format which allows to tag GH users both in VS Code and in GH web UI.